### ISSUE TITLE
(1) Gather collection-level descriptions for already published collections

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#286]
### Descriptive Summary

We would like to add collection-level metadata for collections ingested/published before collection-level metadata was introduced.  This ticket has been created to collect descriptions of these collections so that it can be added once we have determined our strategy for how to best "add" or display it.

### Collection Level Metadata Values

**Harry Mayo**
description:  This collection contains photographs of Santa Cruz Surfing Club members and associated individuals from the 1930s-1940s. Most images capture the surfing scene centered around Cowell Beach in Santa Cruz.

**MS 20:  RMB-PJ**
description: This collection contains a selection of photographs by Ruth-Marion Baruch and Pirkle Jones, whose work visually documents the people and landscape of the San Francisco Bay Area during a time of enormous social, political, and environmental change.

Included here is the entirety of Baruch's work, from her earliest 1940s prints, taken during her time as a college student and novice photographer, to her later works in portraiture as a seasoned artist. Social consciousness permeated Baruch's themes centered around women, children, youth, and race, as evidenced by her most famous projects, such as The Black Panthers, Haight-Ashbury, and Walnut Grove, as well as her lesser-known works like Renaissance Faire, Zoo, and Children, They Grow in the City. In addition to Pirkle Jones' contributions to the Black Panther series of photographs, included here is his collaboration with Dorothea Lange, the photographic series, Death of a Valley. 

_Sensitive content_
Some image titles in this collection contain offensive terms that identify subjects by race, ethnicity, or physical appearance.  These titles were supplied by the artist(s) or were derived from inventories provided by the donor.  In publishing this metadata, we recognize that these terms do not align with current terminology and practice; we chose to retain them to reflect and preserve the context in which these images were created and named.  Some photographs also contain nudity.

**UA 50: Campus History Photographs**
description: The Photography Services Photographs collection includes photographs created by staff photographers employed by the University of California, Santa Cruz. The collection visually documents the history of UCSC from the campus opening in 1965, until the Photography Services department was closed, in 2005. Images include aerial and landscape photographs of the campus and neighboring Santa Cruz, construction views, scenes from daily campus life and activities, classrooms, labs, and curricular materials, and people and events significant to UCSC history.

**Porter College**
description: The Porter College dedication photographs include a scrapbook of photographs and ephemera marking the dedication of Porter College (formerly College Five) at the University of California, Santa Cruz in 1981. Also included are photographs of College Five students and faculty dating from the period.

**UCSC Faculty Lectures**
description: The UCSC Faculty Lectures collection includes recordings of select lectures given by preeminent faculty of the University of California, Santa Cruz.

**Shakespeare SC Posters**
description: The Shakespeare Santa Cruz posters include posters from the 1982-2013 theatrical seasons of Shakespeare Santa Cruz, a professional repertory theater company formerly based in and supported by the University of California, Santa Cruz.

**SC County Historic Photographs**

This collection of photographs spans more than 100 years of Santa Cruz city and county development and activity. The photographs document communities and towns, some now gone; businesses and stores; industries: logging, mining, farming, ranching; the natural surroundings: beaches, forests, rivers, creeks, lagoons; cultural events and entertainment: theater, exhibits, celebrations, parades; institutions: government, churches, schools, libraries; military displays and recreation: team sports, camping, and fishing; and means of transportation: railroads, streetcars, airplanes, automobiles, ships and boats. It reveals the daily lives of Santa Cruz residents and where they lived, worked, played, and worshiped. The bulk of the photographs document the city of Santa Cruz, but there are many images of the North and South county as well. Other large portions include the Santa Cruz Boardwalk and Casino, and the nascent film industry in Santa Cruz including actors, directors, and film crews on location.

Some image titles in this collection contain offensive terms that identify subjects by race, ethnicity, or physical appearance.  These titles were supplied by the artist(s) or were derived from inventories provided by the donor.  In publishing this metadata, we recognize that these terms do not align with current terminology and practice; we chose to retain them to reflect and preserve the context in which these images were created and named.

**Ansel Adams**

This online collection consists of 127 digitized images, selected from the Library's 485 black and white, silver gelatin prints by Ansel Adams. Adams took the photographs between 1962 and 1968; they document the land, building sites, and early activities of the University of California, Santa Cruz campus.

**Cabrillo Festival of Contemporary Music**

The Cabrillo Festival of Contemporary Music (previously Cabrillo Music Festival), is an annual music festival featuring symphonic works by contemporary composers that takes place in Santa Cruz, California. This online collection consists of recordings of concerts as well as accompanying lectures dating from 1964-1990.

### Acceptance Criteria

This is what done looks like:

* [x] Collection level descriptions for each of the collections listed above have been added to this card

### Related Work 

#285 Determine strategy for retroactively adding collection-level information for collections that have already been published

**Comment by rmjaffe** [comment]
@katedundon @tmariamora Could I ask you to supply collection descriptions for the collections listed above? We're going to be working on retrospectively adding this information to the collections that have already been ingested.  I copied descriptions for a couple collections from OAC -- feel free to replace these!!

**Comment by katedundon** [comment]
Here's what I have so far @rmjaffe:

**UA 50: Photography Services Photographs** 
The Photography Services Photographs collection includes photographs created by staff photographers employed by the University of California, Santa Cruz. The collection visually documents the history of UCSC from the campus opening in 1965, until the Photography Services department was closed, in 2005. Images include aerial and landscape photographs of the campus and neighboring Santa Cruz, construction views, scenes from daily campus life and activities, classrooms, labs, and curricular materials, and people and events significant to UCSC history.
(for this one, I wasn't sure if we changed the title of the collection to just Campus History Photographs, or if it is broadened to include images from other campus history collections. If it isn't exclusively UA50, I can revise)

**UA133: Porter College dedication photographs**
The Porter College dedication photographs depict students and faculty of Porter College (formerly College Five) at the dedication of Porter College in 1981.

**UA41: Shakespeare Santa Cruz posters**
The Shakespeare Santa Cruz posters include posters from the 1982-2013 theatrical seasons of Shakespeare Santa Cruz, a professional repertory theater company based in and supported by UC Santa Cruz.

I'm hoping @tmariamora can supply the collection description for the faculty lectures, or point me to some kind of documentation about it.

**Comment by tmariamora** [comment]
It's very basic/ generic, but here you go: The UCSC Faculty Lectures collection includes recordings of select lectures given by preeminent faculty of the University of California, Santa Cruz.

**Comment by rmjaffe** [comment]
@katedundon This description is just for UA 50, so what you've supplied is perfect.  Once we begin the combined campus history collections ingest and the collections-of-collections work, I'll circle back for a description of the whole, including UA 50.

**Comment by tmariamora** [comment]
@rmjaffe @katedundon I've completed a review of the collection descriptions for these items and added the new Sensitive Materials statement to RMB/PJ. I've also done a quick review of the other collections on this card and don't think that any of the other collections need such a statement. @katedundon Can you please review and make sure you're ok with everything?

**Comment by rmjaffe** [comment]
@tmariamora and @katedundon As this card was created (and closed) a while back, I reopened so that we can add collection descriptions for collections that have been ingest in the DAMS have not been represented here: SC Co. History, Ansel Adams, CMF.

**Comment by katedundon** [comment]
@tmariamora These look great. I made a couple of tweaks to the descriptions in the ticket:
- Removed the last sentence in the Mayo description, as it refers to part of the photo series that wasn’t digitized.
- Added “formerly” to the Shakespeare Santa Cruz description.

Questions:
- For MS20, could we replace “a time of enormous social, political, and environmental change” with the “the second half of the 20th century”? I realize this is the text in the finding aid, but it's not very specific and I'm not sure what the function of this statement is. 
- Can we ensure the the title “Cabrillo Festival of Contemporary Music” is reflected on the landing page? (rather than “Cabrillo Music Festival")

**Comment by tmariamora** [comment]
@katedundon All makes sense; I made the change to the MS20 blurb and I agree about "Cabrillo Festival of Contemporary Music". 
@rmjaffe I think the ticket can be closed (again). I'm also wondering if we should schedule a meeting with Jess (?) to make sure we're all on the same page about our expectations for the collection landing pages. I think we may have had a discussion early on but it feels like it may be time to revisit. Thoughts?

**Comment by rmjaffe** [comment]
@tmariamora @katedundon:

The titles given in this card are not the published titles of the collections; rather, they're just the shorthand titles I listed for the sake of indicating which collections were wanting collection descriptions.  The only information this card that will be pulled into the DAMS are the collection descriptions supplied by SCA.  If you login to the DAMS, go to the dashboard, select all collections, find and pop open Cabrillo, you will see that current iteration of the landing page displays the formal title.

Work on the collection landing pages is underway.  Within the current sprint, there are four tickets related to this work:

Apply new Collection landing page design: Desktop #176 
Apply new Collection Landing page design: Mobile #276 
Create Collection landing page for Cabrillo Music Festival #275 
Create Collection Landing Page Description Box #222 

The Dev Team recognizes that this work is a priority for SCA and I totally understand everyone's eagerness to see more progress on the front-end.  That said, any delay on this work are owing to other Library priorities, as well as some issues that cropped up with Ryan's development environment.  It was definitely sometime ago, but if I recall (correct me if I'm wrong @JessWaggs), apart from the initial requirements gathering, we did meet about the collection landing pages.  If you open the tickets listed above, most of them contain mock-ups of what the layout will look like.

As far as scheduling a follow-up meeting about collection landing page expectations, I'd strongly prefer to wait until the Team has been able to wrap up these tickets, at which point we'll definitely be seeking your feedback.  I do not want to complicate or disrupt work that is already underway, and I also think it will aid our conversation to have an actual "product" to look at and discuss.  Thanks for your patience!
